persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -21,8 +25,11 @@ module Database.Persist.Postgresql
     , createPostgresqlPoolWithConf
     , module Database.Persist.Sql
     , ConnectionString
+    , HandleUpdateCollision
```

Comment:
We're not currently exporting any of the constructors for this, nor are we exporting any way to create it.

Let's copy the functions from MySQL so we can do this.

We also need the `EntityField rec typ -> Filter rec` that'll return `False` when the `excluded` column is equal to the table's current column.

Reply:
I was WONDERING why HLS was yelling at me!  Couldn't get the compiler to break locally so thank you for confirming that something was wrong.